hack/deployer/runner/gke.go

```diff
@@ -173,24 +245,46 @@ func (d *GKEDriver) create() error {
 		opts = append(opts, "--create-subnetwork range={{.ClusterIPv4CIDR}}", "--cluster-ipv4-cidr={{.ClusterIPv4CIDR}}", "--services-ipv4-cidr={{.ServicesIPv4CIDR}}")
 	}
 
+	labels, err := d.resourcesLabels()
+	if err != nil {
+		return err
+	}
 	return exec.NewCommand(`gcloud beta container --quiet --project {{.GCloudProject}} clusters create {{.ClusterName}} ` +
-		`--region {{.Region}} --no-enable-basic-auth --cluster-version {{.KubernetesVersion}} ` +
+		`--labels "` + labels + `" --region {{.Region}} --no-enable-basic-auth --cluster-version {{.KubernetesVersion}} ` +
 		`--machine-type {{.MachineType}} --disk-type pd-ssd --disk-size 30 ` +
 		`--local-ssd-count {{.LocalSsdCount}} --scopes {{.GcpScopes}} --num-nodes {{.NodeCountPerZone}} ` +
-		`--enable-stackdriver-kubernetes --addons HorizontalPodAutoscaling,HttpLoadBalancing ` +
```

Comment:
`--enable-stackdriver-kubernetes` is deprecated. I don't think however that removing this flag disable log collection or monitoring, see default logging and monitoring behaviour [here](https://cloud.google.com/stackdriver/docs/solutions/gke/installing#available-logs). I still have some metrics in my GKE dashboard, and my saved queries in the logs explorer are still working (yeah must confess I'm using it from time to time 😄 )

Otherwise, yes, I'm removing it because it's deprecated, it's not related to this PR.